Testscripts/Windows/STOR-VHDX-RESIZE-GROWFS.ps1

```diff
@@ -22,6 +22,36 @@ Param([String] $TestParams)
 $ErrorActionPreference = "Stop"
 $testResult = "FAIL"
 
+# If the vm has more than one disk controller, the order in which their corresponding device nodes are added is arbitrary
+# This may result in device names like /dev/sda and /dev/sdc switching around on each boot
+Function Get-DeviceName
+{
+	param ($ip, $port)
+	$scriptContent = @'
+#! /bin/bash
+devlists=(/dev/sda /dev/sdb /dev/sdc)
```

Comment:
devlist, it's a single list

Reply:
Yes. The total disks are 3 in the tests. The device name must be /dev/sdX(X=a,b,c), but the order maybe different on each boot.